site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -96,6 +96,7 @@ const SubmitForm = () => {
             : values.submitters
           : ['Anonymous'],
         language: 'en',
+        editor_notes: values.editor_notes,
```

Comment:
Not necessary because `editor_notes` is already being set in the spread above (`...values`)